### PR TITLE
[Sage-552] Panel Controls, Toolbar - Alignment Fix

### PR DESCRIPTION
**Pull request by anechol**
## Description

Fixes alignment with components within the Panel Controls and Toolbar.

## Screenshots

|  Before  |  After  |
|--------|--------|
|||
Screen Shot 2022-05-12 at 3 49 08 PM|Screen Shot 2022-05-12 at 3 38 41 PM
Screen Shot 2022-05-12 at 3 49 00 PM|Screen Shot 2022-05-12 at 3 38 50 PM

## Testing in `sage-lib`

- View [Panel Controls](http://localhost:4000/pages/component/panel_controls?tab=preview) and [Toolbar](http://localhost:4000/pages/component/toolbar?tab=preview)
- Check that alignment issues are resolved

## Testing in `kajabi-products`

(**LOW**) Fixes alignment with components within the Panel Controls and Toolbar.

## Related

[Sage-552](https://kajabi.atlassian.net/browse/SAGE-552)

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/themes/next/components/_toolbar.scss [hunk 1]
```diff
@@ -5,6 +5,10 @@
 ///
 /// NOTE: This is an early exploration effort to break away from Panel Controls for creating "toolbar" setups. *?
 
+// Fixes issue with input alignment
+:root {
+  --icon-block-padding: #{rem(4px)};
```
Comment (monkeypox8) [inline packages/sage-assets/lib/stylesheets/themes/next/components/_toolbar.scss:10]: Same here; consider moving under `.sage-toolbar` or `.sage-toolbar__group`

**Review by monkeypox8 — CHANGES_REQUESTED** [review]
Looking great. I've added some comments around the scoping of the css custom props.

**Review by pixelflips — APPROVED** [review]
LGTM! Looks like things are aligning again, nice work! 👍🏼

**Review by monicawheeler — APPROVED** [review]

**Review by QuintonJason — APPROVED** [review]
Looking good

**Pull request merged**